libs/components/docs-tools/src/lib/modules/type-definition/pipes/type-anchor-links.pipe.ts

```diff
@@ -26,47 +26,70 @@ export class SkyTypeAnchorLinksPipe implements PipeTransform {
   readonly #anchorSvc = inject(SkyDocsTypeDefinitionAnchorIdsService);
   readonly #changeDetector = inject(ChangeDetectorRef);
   readonly #destroyRef = inject(DestroyRef);
+  readonly #logSvc = inject(SkyLogService);
   readonly #windowRef = inject(SkyAppWindowRef);
 
-  readonly #cache: Record<string, string> = {};
+  readonly #cache: Record<string, { type: string; anchorEl: string }> = {};
```

Comment:
👇 Added some performance improvements to the anchor link pipe. Only one query params subscription.